lib/cisco_node_utils/interface.rb

```diff
@@ -96,6 +97,22 @@ def default_description
       config_get_default('interface', 'description')
     end
 
+    def enable_pim_sparse_mode
+      state = config_get('interface', 'pim_sparse_mode', @name)
+      state ? true : false
+    end
+
+    def enable_pim_sparse_mode=(state)
+      no_cmd = (state ? '' : 'no')
+      config_set('interface', 'pim_sparse_mode', @name, no_cmd)
+    rescue Cisco::CliError => e
+      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
+    end
```

Comment:
There needs to be a feature pim check:
e.g.

```
def pim_sparse_mode=(state)
  return false unless Pim.feature_enabled
  config_get('interface', 'pim_sparse_mode', @name)
end
```

Smitha's working on the Pim.feature_enabled part. We can either dupe it here temporarily or wait for her code.